.github/workflows/build_pr_container.yaml

```
@@ -1,15 +1,16 @@
 name: Build PR container
 
-on:
-  pull_request:
-    paths-ignore:
-      - ".github/integration/**"
-      - ".github/workflows/**"
-      - ".gitignore"
-      - "**/*.md"
-      - ".github/dependabot.yaml"
-      - "charts/**"
-      - "Makefile"
+#on:
+#  pull_request:
+#    paths-ignore:
+#      - ".github/integration/**"
+#      - ".github/workflows/**"
+#      - ".gitignore"
+#      - "**/*.md"
+#      - ".github/dependabot.yaml"
+#      - "charts/**"
+#      - "Makefile"
+#      - "sda-admin/**"
```

```suggestion
on:
  pull_request:
    paths-ignore:
      - ".github/integration/**"
      - ".github/workflows/**"
      - ".gitignore"
      - "**/*.md"
      - ".github/dependabot.yaml"
      - "charts/**"
      - "Makefile"
      - "sda-admin/**"
```

This should be active in order to be able to run the tests.